runsauce.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
why is this file necessary?

Reply:
This wasnt working when I was using CMD, changed to ENTRYPOINT and rollback to use runsauce.js